### ISSUE TITLE
unsupported VA image format nv12 with Iris mesa driver (19.1.x)

### ISSUE THREAD

**Issue by enihcam** [intel/media-driver#664]
Repro steps:

mesa 19.1.1 with `MESA_LOADER_DRIVER_OVERRIDE=iris`
intel-media-driver 19.1.0 with `LIBVA_DRIVER_NAME=iHD`

mpv with an H264 video, and you shall see no pictures at all, with logs:
```
Using hardware decoding (vaapi).
AO: [pulse] 48000Hz stereo 2ch float
VO: [gpu] 1280x534 => 1281x534 vaapi[nv12]
[vo/gpu/vaapi-egl] unsupported VA image format nv12
[vo/gpu] Initializing texture for hardware decoding failed.
```

**Comment by wangyan-intel** [comment]
@enihcam Could you share your video clips? And what OS you run MPV? Ubuntu? Fedora? Thanks.

**Comment by enihcam** [comment]
@wangyan-intel Unfortunately I cannot share my video clip.

The OS is Archlinux, with all packages sync-ed. Here is the mpv.conf:

```
vo=gpu
hwdec=vaapi
```

**Comment by FurongZhang** [comment]
I saw yan had followed up this issue. So assign it to Yan. Thanks Yan in advance! Maybe related to video decoding.

**Comment by enihcam** [comment]
Related issue:
https://bugzilla.freedesktop.org/show_bug.cgi?id=110666

**Comment by wangyan-intel** [comment]
Based on https://bugzilla.freedesktop.org/show_bug.cgi?id=110666, 

Kenneth Graunke 2019-05-17 02:47:16 UTC
That makes sense.  Commit 4b5e8eb3c8d709bd7c6d1a33a114bf4b002548f8 from today should also help with NV12 support.

We're not backporting iris fixes to 19.1; feel free to use master.
